autoscaler/controllers/gateway/config/root.go

```diff
@@ -38,7 +38,13 @@ func Calculate(dests *odigosv1.DestinationList, processors *odigosv1.ProcessorLi
 			return "", fmt.Errorf("no configer for %s", dest.Spec.Type)
 		}
 
-		configer.ModifyConfig(&dest, currentConfig)
+		err := configer.ModifyConfig(&dest, currentConfig)
+		if err != nil {
+			return "", err.Error()
+		} else {
+			return "Success", nil
```

Comment:
If we return here, the for loop will terminate after the first destination. we need to keep running it for all destinations and allow this function to execute the code below the for loop as well